src/mindlessgen/qm/xtb.py

```diff
@@ -41,8 +44,9 @@ def optimize(
         if np.any(molecule.ati > 85):
```

Comment:
In line above: Shift `ati_original` into `if` clause.

Reply:
Pylint does not like this change because in principle later the `ati_original` variable could be used when it is not initialized.

Reply:
If it's the exactly same `if` condition, just put the appropriate `pylint` ignore message as a comment after the line where the error is given, like
```python
# pylint: disable=no-member
```